cheatsheets/Android_Codenames_Tags_Build_Numbers.rb

```diff
@@ -9,6 +9,16 @@
   category do
     id 'API Levels'
 
+    entry do
+      command '28'
+      name '9.0.0'
```

Comment:
Why `9.0.0` instead of `9.0`?

Reply:
Just because 8.1.0 is listed on the official doc and I was keeping to full three digit precision.

Looking at the actual docs they only specify 9 so perhaps that would be the better way to go.

https://source.android.com/setup/start/build-numbers

Reply:
I'll change it to 9, so it's the same as that page.